connectors/src/connectors/confluence/index.ts

```diff
@@ -237,50 +229,41 @@ export async function resumeConfluenceConnector(
 export async function cleanupConfluenceConnector(
   connectorId: ModelId
 ): Promise<Result<undefined, Error>> {
-  const connector = await ConnectorModel.findOne({
-    where: { type: "confluence", id: connectorId },
-  });
+  const connector = await ConnectorResource.fetchById(connectorId);
   if (!connector) {
     logger.error({ connectorId }, "Confluence connector not found.");
     return new Err(new Error("Connector not found"));
   }
 
-  return sequelizeConnection.transaction(async (transaction) => {
-    await Promise.all([
-      ConfluenceConfiguration.destroy({
-        where: {
-          connectorId: connector.id,
-        },
-        transaction,
-      }),
-      ConfluenceSpace.destroy({
-        where: {
-          connectorId: connector.id,
-        },
-        transaction,
-      }),
-      ConfluencePage.destroy({
-        where: {
-          connectorId: connector.id,
-        },
-        transaction,
-      }),
-    ]);
+  await Promise.all([
+    ConfluenceConfiguration.destroy({
+      where: {
+        connectorId: connector.id,
+      },
+    }),
+    ConfluenceSpace.destroy({
+      where: {
+        connectorId: connector.id,
+      },
+    }),
+    ConfluencePage.destroy({
+      where: {
+        connectorId: connector.id,
+      },
+    }),
+  ]);
 
-    const nangoRes = await nangoDeleteConnection(
-      connector.connectionId,
-      getRequiredNangoConfluenceConnectorId()
-    );
-    if (nangoRes.isErr()) {
-      throw nangoRes.error;
-    }
+  const nangoRes = await nangoDeleteConnection(
```

Comment:
Will this be part of the resource too ? or is external to the resource but still closely tied to it 🤔  ?